Makefile

```diff
@@ -2,7 +2,7 @@ DESCRIPTION = CS50 Library for Java
 MAINTAINER = CS50 <sysadmins@cs50.harvard.edu>
 OLD_NAME = lib50-java
 NAME = libcs50-java
-VERSION = 2.0.0
+VERSION = 2.0.1
```

Comment:
@dmalan, we haven't actually released v2.0.0 yet. Perhaps we shouldn't up that now?